chia/types/eligible_coin_spends.py

```diff
@@ -354,4 +354,4 @@ async def process_fast_forward_spends(
         # change. Still, it's good form to update the spend bundle with the
         # new coin spends
         mempool_item.spend_bundle = new_sb
-        mempool_item.conds = new_npc_result.conds
+        mempool_item.conds = new_sbc_result
```

Comment:
With a name like `new_conditions` or `new_conds` this reads better.

Reply:
Done